custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -942,6 +943,21 @@ def _prepare_docker_command(self, options, run_mode, raw_arguments):
             )
             docker_cmd_args += " -p {port}:{port}".format(port=port)
 
+        if "runtime_params_file" in options and options.runtime_params_file:
```

Comment:
this is extremely suspicious.  don't we set all of this stuff in `enums.py`?

Reply:
see `datarobot_drum.drum.args_parser.CMRunnerArgsRegistry._reg_arg_custom_task_user_secrets` for an example.

also see: `unit.datarobot_drum.drum.test_args_parser.TestUserSecretsArgs.test_set_user_secrets_from_env_vars`

Reply:
keeping this unresolved because the similar code in `main.py` would be worth addressing in the future